Project.toml

```diff
@@ -50,6 +50,8 @@ julia = "1.5"
 [extras]
 AlgebraicMultigrid = "2169fc97-5a83-5252-b627-83903c6c433c"
 Calculus = "49dc2e85-a5d0-5ad3-a950-438e2897f1b9"
+DiffEqFlux = "aae7a2af-3d4f-5e19-a356-7da93b79d9d0"
```

Comment:
This should get a separate test section with a separate Project.toml for downstream. See https://github.com/SciML/DiffEqBase.jl/blob/master/test/runtests.jl#L38-L40 as an example.